curvefs/src/client/inode_wrapper.h

```diff
@@ -80,28 +82,40 @@ class InodeWrapper : public std::enable_shared_from_this<InodeWrapper> {
     InodeWrapper(const Inode &inode,
                  const std::shared_ptr<MetaServerClient> &metaClient,
                  const std::shared_ptr<S3ChunkInfoMetric>
-                    &s3ChunkInfoMetric = nullptr)
+                    &s3ChunkInfoMetric = nullptr,
+                 uint64_t maxDataSize = ULONG_MAX,
+                 uint32_t refreshDataInterval = UINT_MAX)
         : inode_(inode),
           status_(InodeStatus::Normal),
           isNlinkValid_(true),
           metaClient_(metaClient),
           s3ChunkInfoMetric_(s3ChunkInfoMetric),
           openCount_(0),
-          dirty_(false) {
+          dirty_(false),
+          baseMaxDataSize_(maxDataSize),
+          maxDataSize_(maxDataSize),
+          refreshDataInterval_(refreshDataInterval),
+          lastRefreshTime_(0) {
```

Comment:
it's better to set it to current timestamp

Reply:
fixed